packages/cypress/tests/cypress/e2e/custom-downloads-directory.cy.ts

```diff
@@ -0,0 +1,7 @@
+context('using Cypress.config', () => {
+  it('Downloads archives to the user-specified folder', () => {
+    cy.visit('/viewports');
+    const dirExists = cy.task('directoryExists', '/some-dir');
+    console.log('DIRECTORY EXISTS', dirExists);
+  });
+});
```

Comment:
I'm not as familiar with the Cypress workflow as I'd like to be. @skitterm can you briefly describe what this does and how it might help verify that custom downloads folders now work for Cypress?

Reply:
@andrewortwein good point -- this commit was a bare-bones pipeline for @tevanoff to verify that the downloads folder exists. 

In Cypress, the code in these test files is run in the browser (not in Node like Playwright). In order to access the file system, Cypess has a concept of "[tasks](https://docs.cypress.io/api/commands/task)" that run code on a Node backing. The intent here was to call this task (`directoryExists`) which would read the file system and verify the custom downloads directory exists.

However, this test is looking to be cumbersome to write, so I [believe](https://github.com/chromaui/chromatic-e2e/pull/164#issuecomment-2190049552) we want to forgo testing this. @tevanoff, feel free to revert the commit where I added this pipeline!